### PR TITLE
feat: move Frax out of isolation mode on mainnet

### PR DESCRIPTION
**Pull request by Rozengarden**

### CODE REVIEW

File: src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/RemoveFraxFromIsolationModeOnAaveV3Mainnet.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+| Borrowable in Isolation  | NO             |
+| Stable Borrow            | NO             |
+| Enable Borrow            | NO             |
+| Enable Collateral        | YES            |
+| Loan To Value            | 0%             |
+| Liquidation Threshold    | 75.00%         |
+| Liquidation Bonus        | 6.00%          |
+| Reserve Factor           | 10.00%         |
+| Liquidation Protocol Fee | 20.00%         |
+| Supply Cap(sFRAX)        | 10.00M         |
```
Comment (brotherlymite) [inline src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/RemoveFraxFromIsolationModeOnAaveV3Mainnet.md:37]:
```suggestion
| Supply Cap        | 10.00M         |
```

File: src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/RemoveFraxFromIsolationModeOnAaveV3Mainnet.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+| Borrow Cap(sFRAX)        | N/A            |
```
Comment (brotherlymite) [inline src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/RemoveFraxFromIsolationModeOnAaveV3Mainnet.md:38]:
```suggestion
| Borrow Cap        | 10.00M            |
```

File: src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/RemoveFraxFromIsolationModeOnAaveV3Mainnet.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+| Uoptimal                 | N/A            |
```
Comment (brotherlymite) [inline src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/RemoveFraxFromIsolationModeOnAaveV3Mainnet.md:39]: I would not include interest rate in the table as they are not changing

File: src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/RemoveFraxFromIsolationModeOnAaveV3Mainnet.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+- FRAX will be removed from Isolation Mode on Aave v3 instances.
+- FRAX will receive same parameters as DAI, a Reserve Factor as other stables, and LTV of 0%.
+
+| Parameter                | Recommendation |
+| ------------------------ | -------------- |
+| Isolation Mode           | NO             |
+| Borrowable in Isolation  | NO             |
+| Stable Borrow            | NO             |
+| Enable Borrow            | NO             |
```
Comment (brotherlymite) [inline src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/RemoveFraxFromIsolationModeOnAaveV3Mainnet.md:30]:
```suggestion
| Enable Borrow            | YES             |
```

File: src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/RemoveFraxFromIsolationModeOnAaveV3Mainnet.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+| Enable Collateral        | YES            |
```
Comment (brotherlymite) [inline src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/RemoveFraxFromIsolationModeOnAaveV3Mainnet.md:31]: Tricky to say it is enabled as collateral as LTV is 0

Comment (Rozengarden) [reply src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/RemoveFraxFromIsolationModeOnAaveV3Mainnet.md:31]: I believe enable borrow & collat got mixed up, I left it as is while I double check with risk

File: src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet_20241105.sol [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    pure
+    override
+    returns (IAaveV3ConfigEngine.CollateralUpdate[] memory)
+  {
+    IAaveV3ConfigEngine.CollateralUpdate[]
+      memory collateralUpdate = new IAaveV3ConfigEngine.CollateralUpdate[](1);
+
+    collateralUpdate[0] = IAaveV3ConfigEngine.CollateralUpdate({
+      asset: AaveV3EthereumAssets.FRAX_UNDERLYING,
+      ltv: 0,
```
Comment (brotherlymite) [inline src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet_20241105.sol:28]:
```suggestion
      ltv: EngineFlags.KEEP_CURRENT,
```
LTV for frax is already 0, so we don't need to change

File: src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet_20241105.sol [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+      liqThreshold: EngineFlags.KEEP_CURRENT,
+      liqBonus: EngineFlags.KEEP_CURRENT,
+      debtCeiling: 0,
+      liqProtocolFee: EngineFlags.KEEP_CURRENT
+    });
+
+    return collateralUpdate;
+  }
+  function borrowsUpdates()
```
Comment (brotherlymite) [inline src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet_20241105.sol:37]:
```suggestion
  function borrowsUpdates()
```

File: src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet_20241105.sol [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+      asset: AaveV3EthereumAssets.FRAX_UNDERLYING,
+      enabledToBorrow: EngineFlags.KEEP_CURRENT,
+      flashloanable: EngineFlags.KEEP_CURRENT,
+      borrowableInIsolation: EngineFlags.KEEP_CURRENT,
+      withSiloedBorrowing: EngineFlags.KEEP_CURRENT,
+      reserveFactor: EngineFlags.KEEP_CURRENT
+    });
+
+    return borrowUpdates;
+  }
```
Comment (brotherlymite) [inline src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet_20241105.sol:56]:
```suggestion
```

File: src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/RemoveFraxFromIsolationModeOnAaveV3Mainnet.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
+---
+title: "Remove Frax from Isolation Mode on Aave v3 Mainnet"
+author: "Aave Chan Initiative"
+discussions: "https://governance.aave.com/t/arfc-remove-frax-from-isolation-mode-on-aave-v3-mainnet/19337"
+snapshot: "https://snapshot.org/#/aave.eth/proposal/0x9bc3f3d8e38d70f55887f2f2498e1b39f59467489158923488aceab73cd4f144"
+---
+
+## Simple Summary
+
+This is an ARFC to remove FRAX from isolation mode, and set LTV to 0%. This is a reboot with minor changes of a [previous ARFC](https://governance.aave.com/t/arfc-remove-frax-from-isolation-mode-and-onboard-sfrax-to-aave-v3-mainnet/18506) which passed Snapshot but has not yet been implemented.
```
Comment (brotherlymite) [inline src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/RemoveFraxFromIsolationModeOnAaveV3Mainnet.md:10]:
```suggestion
This is an AIP to remove FRAX from isolation mode. This is a reboot with minor changes of a [previous ARFC](https://governance.aave.com/t/arfc-remove-frax-from-isolation-mode-and-onboard-sfrax-to-aave-v3-mainnet/18506) which passed Snapshot but has not yet been implemented.
```

File: src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/RemoveFraxFromIsolationModeOnAaveV3Mainnet.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+
+## Motivation
+
+FRAX and Aave DAO have found more synergies over the last months. The FRAX team has responded with major updates to security on sfrxETH in response to BGD Labs feedback. FRAX has also initated governance proposals to add GHO to Frax Lend. There are ongoing conversations to have a FRAX AMO included into Aave v3. sFRAX was [previously accepted](https://governance.aave.com/t/arfc-add-sfrax-on-ethereum-v3/16303) for onboarding in a previous [ARFC vote](https://snapshot.org/#/aave.eth/proposal/0xdba99e9c8da24424447d7c7b70eff93ad5b6055714b5f34cf9859c923fb3a38a) before the introduction of CAPO feeds.
+
+This proposal suggests removing FRAX from isolation mode to facilitate further AMO deployments.
+
+Furthermore, this proposal sets LTV to 0% as there is no use-case for FRAX to act as collateral.
```
Comment (brotherlymite) [inline src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/RemoveFraxFromIsolationModeOnAaveV3Mainnet.md:18]:
```suggestion
```

File: src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/RemoveFraxFromIsolationModeOnAaveV3Mainnet.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+
+## Specification
+
+- FRAX will be removed from Isolation Mode on Aave v3 instances.
+- FRAX will receive same parameters as DAI, a Reserve Factor as other stables, and LTV of 0%.
```
Comment (brotherlymite) [inline src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/RemoveFraxFromIsolationModeOnAaveV3Mainnet.md:23]:
```suggestion
```
RF of FRAX is 20% while DAI is 25%

File: src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/RemoveFraxFromIsolationModeOnAaveV3Mainnet.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+
+| **Parameter**  | FRAX |
+| -------------- | ---- |
+| Isolation Mode | No   |
+| LTV            | 0%   |
```
Comment (brotherlymite) [inline src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/RemoveFraxFromIsolationModeOnAaveV3Mainnet.md:28]:
```suggestion
```

File: src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/RemoveFraxFromIsolationModeOnAaveV3Mainnet.md [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+| Debt Ceiling   | -    |
```
Comment (brotherlymite) [inline src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/RemoveFraxFromIsolationModeOnAaveV3Mainnet.md:29]:
```suggestion
| Debt Ceiling   | 0    |
```

**Review by brotherlymite — APPROVED** [review]

**Review by ianflexa — APPROVED** [review]

**Pull request merged**